server/routes/api/auth/authorize.js

```diff
@@ -0,0 +1,38 @@
+import { Router} from 'express';
+import Authorization from '../../../controllers/Authorization';
+import RoleAuthorization  from '../../../middlewares/RoleAuthorization';
+import JWTAuthentication from '../../../middlewares/JWTAuthentication';
+
+const router = Router();
+
```

Comment:
superAdmin should be put in a variable for reusability.
So, if you needed to change the superAdmin string, you only need to change it once instead of all the places it appears

Reply:
@theghostyced here. Hi @JohnMadakin. I totally agree with you on this. Feedbacks would be done ASAP. Thanks for pointing this out.